test/unit/Governor.js

```diff
@@ -0,0 +1,444 @@
+import Fixture from "./helpers/Fixture"
+
+import truffleAssert from "truffle-assertions"
+import {utils} from "ethers"
+
+const Governor = artifacts.require("Governor")
+const ECTools = artifacts.require("ECTools")
+const SetUint256 = artifacts.require("SetUint256")
+
+contract("Governor", accounts => {
+    let fixture
+    let governor
+    let ecTools
+    let setUint256
+
+    before(() => {
+        fixture = new Fixture(web3)
+    })
+
+    beforeEach(async () => {
+        await fixture.setUp()
+        ecTools = await ECTools.new()
+        await Governor.link("ECTools", ecTools.address)
```

Comment:
Looks like we can remove all references to ECTools.

Reply:
fixed in cd04b83